packages/ui/Scrollable/Scrollbar.js

```diff
@@ -92,6 +90,7 @@ const ScrollbarBase = memo(forwardRef((props, ref) => {
 				scrollThumbPositionRatio = (scrollOrigin / (scrollContentSize - scrollViewSize)),
 				scrollThumbSizeRatio = Math.max(ri.scale(minThumbSize) / trackSize, Math.min(1, thumbSizeRatioBase));
 
+			setCSSVariable(thumbRef.current, '--scroll-thumb-min-size', minThumbSize);
```

Comment:
Should we do ri.scale here for `minThumbSize`?

Reply:
I noticed this css variable is not needed. I'm going to remove it .Thanks.